package/network/utils/comgt/files/ncm.sh

```diff
@@ -126,18 +137,27 @@ proto_ncm_setup() {
 		json_select ..
 	}
 
-	json_get_vars connect
-	eval COMMAND="$connect" gcom -d "$device" -s /etc/gcom/runcommand.gcom || {
-		echo "Failed to connect"
-		proto_notify_error "$interface" CONNECT_FAILED
-		return 1
-	}
+#	json_get_vars connect
+#	eval COMMAND="$connect" gcom -d "$device" -s /etc/gcom/runcommand.gcom || {
+#		echo "Failed to connect"
+#		proto_notify_error "$interface" CONNECT_FAILED
+#		return 1
+#	}
 
```

Comment:
You are right, this change require an explanation.
The connection cmd need to be sent after the interface is enabled.
This is the reason why I moved the section after the:
```
proto_init_update "$ifname" 1
proto_send_update "$interface"
```
I will show the failure log in the next day.